helmfile/values/kured.yaml.gotmpl

```diff
@@ -15,4 +15,21 @@ configuration: {{- toYaml .Values.kured.configuration | nindent 2 }}
 
 extraArgs: {{- toYaml .Values.kured.extraArgs | nindent 2 }}
 
+extraEnvVars:
+{{- if .Values.kured.extraEnvVars }}
+{{- toYaml .Values.kured.extraEnvVars | nindent 2 }}
+{{- end }}
```

Comment:
Do you think we'll need this?

Reply:
I added it to future proof it. The only envs i see us setting are from secrets. But you never know :slightly_smiling_face:

Reply:
Fair enough ;)